core/settings.py

```
@@ -0,0 +1,652 @@
+    )
+    AWS_S3_ENDPOINT_URL = "https://nyc3.digitaloceanspaces.com"
+    AWS_REGION_NAME = "nyc3"
+    AWS_S3_OBJECT_PARAMETERS = {
+        "CacheControl": "max-age=86400",
+    }
+    AWS_LOCATION = "static"
+    STATIC_URL = f"{AWS_S3_ENDPOINT_URL}/{AWS_LOCATION}/"
+    STATICFILES_DIRS = [
+        os.path.join(BASE_DIR, "the-dozens-django", "static"),
+    ]
+    STATIC_ROOT = "/tmp/staticfiles"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/staticfiles"

(S108)